src/diffusers/pipelines/oms/__init__.py

```diff
@@ -0,0 +1,71 @@
+from typing import TYPE_CHECKING
+
+from ...utils import (
+    DIFFUSERS_SLOW_IMPORT,
+    OptionalDependencyNotAvailable,
+    _LazyModule,
+    get_objects_from_module,
+    is_flax_available,
+    is_k_diffusion_available,
+    is_librosa_available,
+    is_note_seq_available,
+    is_onnx_available,
+    is_torch_available,
+    is_transformers_available,
+)
+
+_dummy_objects = {}
+
+_import_structure = {
+    "oms_pipeline": [],
+    "cloth_adapter" : []
+}
+
+
+try:
+    if not (is_torch_available() and is_transformers_available()):
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    from ...utils import dummy_torch_and_transformers_objects
+
+
+    _dummy_objects.update(get_objects_from_module(dummy_torch_and_transformers_objects))
+else:
+    _import_structure["oms_pipeline"].extend(
+        [
+            "OmsDiffusionPipeline",
+        ]
+    )
+
+    _import_structure["cloth_adapter"].extend(
+        [
+            "ClothAdapter",
+        ]
+    )
+if TYPE_CHECKING or DIFFUSERS_SLOW_IMPORT:
+    try:
+        if not (is_torch_available() and is_transformers_available()):
+            raise OptionalDependencyNotAvailable()
+    except OptionalDependencyNotAvailable:
+        from ...utils.dummy_torch_and_transformers_objects import *
+    else:
+        from .oms_diffusion_pipeline import (
+            OmsDiffusionPipeline,
+        )
+        from .cloth_adapter import (
```

Comment:
try getting everything into oms_diffusion_pipeline, I don't think so we have to import cloth_adapter separately for this